.github/workflows/ids-check.yml

```diff
@@ -0,0 +1,110 @@
+name: "Check LLVM ABI annotations"
+on:
+  pull_request:
+    branches:
```

Comment:
Why are you specifying branches? We typically don't for our workflows. This also prevents the action from running on PRs into the release branch which may be unintentional.